css/index.css

```
@@ -0,0 +1,6 @@
+@import "./variables.css";
+@import "./card.css";
+@import "./global.css";
```

To avoid conflicts, I would suggest to import the global.css before you import the CSS-files of your components.
```suggestion
@import "./global.css";
@import "./card.css";
```

---

Changed. Thanks. I thought its only important for the variable.css to come first.